pyproject.toml

```
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "chat-with-your-data-solution-accelerator"
+version = "0.1.0"
+description = ""
+authors = []
+readme = "README.md"
+packages = [
+    { include = "code" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+azure-functions = "1.18.0"
+streamlit = "1.30.0"
```

I think this should be 
```suggestion
streamlit = "1.32.0"
```
based on the current requirements.txt